ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestconnectivity.py

```diff
@@ -202,36 +202,44 @@ def test_definition(self) -> dict | None:
 
         required_vars = ["name", "shutdown", "peer", "peer_interface"]
 
-        for ethernet_interface in ethernet_interfaces:
+        for idx, ethernet_interface in enumerate(ethernet_interfaces, start=1):
             try:
                 for var in required_vars:
                     get(ethernet_interface, var, required=True)
+            except AristaAvdMissingVariableError as e:
+                LOGGER.warning("Ethernet interface entry #%d from the 'ethernet_interfaces' data model is missing the variable '%s'.", idx, str(e))
+                continue
```

Comment:
You know the primary key is "name" here, so presenting the value of "name" would be much more helpful to the user instead of the index.

Reply:
It covers if `name` is missing from the structured_config.

Reply:
I agree, that will clean-up a lot of code. Issue #3259 opened. I will work on that later but for now let's guard everything in code.